code/obj/artifacts/artifact_objects/bombs.dm

```diff
@@ -85,12 +85,13 @@ ABSTRACT_TYPE(/datum/artifact/bomb)
 
 			// actual boom
 			SPAWN_DBG(10 SECONDS)
-				if (src.activated)
+				if (O.loc && src.activated)
```

Comment:
i feel like checking .disposed is your intention

Reply:
Ah, so that's how you're supposed to do that, thanks!^^